src/App.jsx

```diff
@@ -1,44 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState('1');
```

Comment:
You should save tab in state, not id
```suggestion
  const [selectedTab, setSelectedTab] = useState(tabs[0]);
```
![2024-04-04_16h26_29](https://github.com/mate-academy/react_tabs-js/assets/61904995/cbab7bc0-4262-48c8-b4f8-5e12c14a4ef6)